lib/DDG/Spice/RandWord.pm

```diff
@@ -15,19 +15,29 @@ icon_url "/i/wordnik.com.ico";
 attribution web => ['http://dylansserver.com','Dylan Lloyd'],
             email => ['dylan@dylansserver.com','Dylan Lloyd'];
 
-spice from => '(?:([0-9]+)\-([0-9]+)|)';
-spice to => 'http://api.wordnik.com/v4/words.json/randomWord?minLength=$1&maxLength=$2&api_key={{ENV{DDG_SPICE_WORDNIK_APIKEY}}}&callback={{callback}}';
+spice from => '(?:([0-9]+)\-([0-9]+)\-([0-9]+))';
+spice to => 'http://api.wordnik.com/v4/words.json/randomWords?minLength=$1&maxLength=$2&limit=$3&api_key={{ENV{DDG_SPICE_WORDNIK_APIKEY}}}&callback={{callback}}';
 spice proxy_cache_valid => "418 1d";
 
-triggers any => "random word";
+triggers any => "random word", "random words";
 
 handle remainder => sub {
-    if ($_ =~ /^([0-9]+\-[0-9]+)$/) {
-        return $1;
-    } elsif ($_ eq '') {
-        return '0-100';
+    my $minlen = 1;
+    my $maxlen = 100;
+    my $limit = 1;
+
+    if ($_ =~ /^([0-9]+)\-([0-9]+)$/) {
+        # Quesries like "random word 5-10"
+        $minlen = $1;
+        $maxlen = $2;
+    } elsif ($_ =~ /^\d+$/) {
+        # Queries like "5 random words"
+        $limit = $_;
```

Comment:
Is there a limit on this from the API? We may want to give it an upper-bound like 20. I haven't checked to see how that looks yet...